models/eos/interfaces/replaced_example_01.txt

```diff
@@ -0,0 +1,43 @@
+
+# Using replaced
+
+# Before state:
+# -------------
+#
+# veos#show running-config | section interface
+# interface Ethernet1
+#    description "Interface 1"
+# !
+# interface Ethernet2
+# !
+# interface Loopback1
+# !
+# interface Management1
```

Comment:
When we have a replaced example, can we add a attribute to one interface that will not be affected by the replaced operation?  In this case, maybe a description for Management1

Reply:
^^^  This might help demonstrate the difference between overridden and replaced.